tensorflow/compiler/xla/pjrt/gpu/BUILD

```diff
@@ -51,8 +51,8 @@ cc_library(
         "//tensorflow/compiler/xla/stream_executor:device_mem_allocator",
         "//tensorflow/compiler/xla/stream_executor:device_memory",
         "//tensorflow/compiler/xla/stream_executor:tf_allocator_adapter",
-    ] + if_cuda([
         ":nccl_id_store",
```

Comment:
nit: can you move this to the top (under `:gpu_helpers`)

Reply:
done